src/constants/constants.js

```diff
@@ -1,2 +1,11 @@
 export const BASE_URL = 'http://localhost:3500/';
-export const OPTIMIZE_SVG= 'optimizeSvg';
\ No newline at end of file
+export const OPTIMIZE_SVG= 'optimizeSvg';
+export const SVG_TAG_NAMES= [
+    "rect",
+    "path",
+    "circle",
+    "polygon",
```

Comment:
remove this from your PR please, src/image/downloadbutton.png , if you want to display an icon please use a cdn